tests/scripts/pdbe_api_status.py

```diff
@@ -0,0 +1,51 @@
+from dabeplech import PDBeAPI
+
+from utils import _format_print
+
+
+def _expected_working_pdbe():
+    tests = [
+        ('Uniprot Mappings', 'mappings/uniprot/3u85'),
```

Comment:
Since you created two classes `PDBeUniprotMappingAPI` and `PDBePFAMMappingAPI` classes, you could directly use them to have something like:

```python
tests = [
    ('Uniprot Mappings', '3u85', 'PDBeUniprotMappingAPI'),
    ('PFAM Mappings', '3u85', 'PDBePFAMMappingAPI'),
]
for i in tests:
    db = i[0]
    entry_id = i[1]
    api_connector = i[2]
    print(f"Test getting {entry_id} from {db} database from base URL: {PDBeAPI.BASE_URL}:")
    try:
        api = api_connector()
        [...]
```

The structure I suggested can be different of course, whatever makes more sense in term of execution of the tested endpoints.